src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -720,7 +720,9 @@
 
   <ItemGroup Condition=" '$(UsesHelixSdk)' == 'true' ">
     <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
-    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" />
+
+    <!-- Browser-Wasm follows a very different workflow, which is currently out of scope of the Log Checker. -->
+    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" Condition="'$(TargetsBrowser)' != 'true'" />
```

Comment:
Does this change the current behavior for wasm logs then?

Reply:
Not at all. Wasm is out of scope of this project for the time being, so wasm tests will remain unaffected.